hw/ip/usbdev/usbdev.core

```diff
@@ -3,7 +3,7 @@ CAPI=2:
 # Licensed under the Apache License, Version 2.0, see LICENSE for details.
 # SPDX-License-Identifier: Apache-2.0
 name: "lowrisc:ip:usbdev:0.1"
-description: "usbuart"
+description: "USB Device"
```

Comment:
Haha, so those are the origins of this device! ;)

Reply:
Apparently so :)...